doc/classes/ProjectSettings.xml

```diff
@@ -2658,6 +2658,21 @@
 		</member>
 		<member name="rendering/rendering_device/staging_buffer/texture_upload_region_size_px" type="int" setter="" getter="" default="64">
 		</member>
+		<member name="rendering/rendering_device/vsync/frame_queue_size" type="int" setter="" getter="" default="2">
+			The number of frames to track on the CPU side before stalling to wait for the GPU.
+			1 may give you the lowest lag/latency but at the high cost of no parallelism between CPU &amp; GPU.
+			Try the [url=https://darksylinc.github.io/vsync_simulator/]V-Sync Simulator[/url], an interactive interface that simulates presentation to better understand how it is affected by different variables under various conditions.
+			[b]Note:[/b] This property is only read when the project starts. There is currently no way to change this value at run-time.
+		</member>
+		<member name="rendering/rendering_device/vsync/swapchain_image_count" type="int" setter="" getter="" default="3">
+			The number of images the swapchain will consist of (back buffers + front buffer).
+			[code]2[/code] corresponds to double-buffering and [code]3[/code] to triple-buffering. A value of [code]1[/code] is not recommended.
```

Comment:
> A value of [code]1[/code] is not recommended.

Why? If it can be summarised with "It just does not look good" feel free to mention it. If the explanation is too long I'm okay with omitting it.

Reply:
The whole block of text has gone through multiple revisions.

The problem is that this setting is related to the following 3 topics:

 - V-Sync
 - Input to Display Latency
 - CPU & GPU synchronization

One could write a 100-page (or more) essay of these 3. There is only so much that can be squashed into a tooltip. They are all advanced topics.

This is why the tooltip refers to the [Simulator](https://darksylinc.github.io/vsync_simulator) which contains an interactive simulator and [links](https://askubuntu.com/a/1420575/274245) to more [bibliography](https://www.gdcvault.com/play/1026327/Controller-to-Display-Latency-in) about it that go in depth.

Regarding this particular question. A value of 1is not recommended, is because the CPU will always have to wait for the GPU, which can hurt performance a lot, but might be important for specific scenarios where latency must be the lowest (real time medical imagining _might_ need this), or RAM is tight and synchronization is not necessary or rare (AI and blockchain mining comes to mind).

However I saw that this PR from @DarioSamo always performs a `actual_count = max( 2u, swapchain_image_count )` unless local device is used. I don't know exactly why he did this, but it may make this comment irrelevant and maybe `swapchain_image_count` should have 2 as min value.

Reply:
We consulted with @reduz and we should just have two as the safe minimum, so the note will be pretty irrelevant.